### PR TITLE
ENCD-5285-table-lines

### PR DESCRIPTION
**Pull request by forresttanaka**
Added a supremely complicated condition for drawing the heavy bar below certain rows. I also fixed one outdated colSpan.

### CODE REVIEW

**Comment by forresttanaka** [comment]
Demo: ~https://encd-5285-table-lines-pr1-fytanaka.demo.encodedcc.org/~

**Comment by forresttanaka** [comment]
Demo for PR: https://encd-5285-table-lines-pr1-fytanaka.demo.encodedcc.org/experiments/ENCSR141MUD/

File: src/encoded/static/components/filegallery.js [hunk 1]
```diff
@@ -670,7 +673,15 @@ class RawSequencingTable extends React.Component {
                                         // Must be an index reads file if it's not paired-ended.
                                         pairClass = 'index-reads';
                                     }
-                                    if (groupFiles[i + 1] && groupFiles[i + 1].pairSortId !== file.pairSortId) {
+
+                                    // Draw bottom border if second file of pair and no index reads
+                                    // follow, or file *is* index reads, but not if last row of
+                                    // table.
+                                    const isNextInGroupIndexReads = groupFiles[i + 1] && groupFiles[i + 1].output_type === 'index reads';
+                                    if (
```
Comment (zoldello) [inline src/encoded/static/components/filegallery.js:681]: Would it be a good idea to place each condition in the if-statement in a variable and use the if-statement on those variables? That has the advantage of keeping the readers' attention focus and may help reduce the length. That is something I picked up from Python.

**Review by zoldello — APPROVED** [review]
Code looks fine. I saw nothing posing a security threat or that looks improper. My comment on the PR is nothing significant and can be ignored if so desired.

**Pull request merged**